cpp/CMakeLists.txt

```diff
@@ -74,20 +77,82 @@ if(ARCTICDB_SANITIZER_FLAGS)
     message(STATUS "Building ArcticDB with sanitizers. Compiler flags: ${ARCTICDB_SANITIZER_FLAGS}")
 endif()
 
+if(${ARCTICDB_USING_CONDA})
+    # Required to be able to include headers from glog since glog 0.7
+    # See: https://github.com/google/glog/pull/1030
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGLOG_USE_GLOG_EXPORT")
```

Comment:
I think this is for setting C macro. I prefer using `add_compile_definitions` or even better `target_compile_definitions` rather than changing the CMAKE_CXX_FLAGS

Reply:
Yes, I agree. For now, I just minimalistically moved code around to have it work, but eventually all the complexity of `CMakeLists.txt` must be removed.